bcu.c

```diff
@@ -1659,8 +1659,16 @@ static void monitor(struct options_setting* setting)
 					}
 					struct gpio_device* gd = end_point;
 					unsigned char data;
-					
-					// set_gpiod(gd, 1); //initialize all pins to 1
+
+					//initialize gpio pin with value from 4 LSB bits of initinfo; 0 means high current, 1 means low current
+					if ((board->mappings[get_gpio_id(sr_name, board)].initinfo & 0xF) == 0)
+					{
+						set_gpiod(gd, 1);
+					}
+					else if ((board->mappings[get_gpio_id(sr_name, board)].initinfo & 0xF) == 1)
+					{
+						set_gpiod(gd, 0);
+					}
```

Comment:
Can I add this separate option, [-init], under monitor command? I need to initialized the range of the probe when we collect the data under some automatic tests when I cannot reset the board or change manually the range. 

I wrote this part of code because I saw that the SR_ pins are not initialized anywhere. That means at startup they can take any value. Sorry if I'm wrong.

Reply:
Yes, you can add [-init] option to deal this part of code.
On the previous i.MX board, those pins controlling the range have default values set through pull-up/pull-down resistors.  So we do not need initialize those pins.

Reply:
I added the -init option. Thank you!